.github/ISSUE_TEMPLATE/funding_application.yml

```diff
@@ -0,0 +1,221 @@
+---
+name: TI Funding Request Form
+description: TI Funding Request Form OpenSSF
+title: "[Technical Initiative Funding Request]: "
+labels:
+  - administration
+  - For Review
+  - Funding Request
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for taking the time to fill out this Funding Request.
+  - type: markdown
+    attributes:
+      value: |
+        **Pre-conditions:**
+        - The TI must have an identified lifecycle phase inside the OpenSSF, such as sandbox, incubating, or graduated. 
+        - The TI requesters should know 
+          - Within general funding limits for the TIs at the respective lifecycle, that are to be determined. 
+          - Within approved categories of asks (e.g. no asking for a TI outing at a fancy restaurant but yes to cloud credits)
+  - type: markdown
+    attributes:
+      value: |
+        **Workflow:**
+        1. TI or TIs working together see a need for funding some work that doesn’t exist within an existing give/get that is pre-approved. For example cloud credits to do a PoC or case study.
+        2. TI(s) get consensus on the scope of how much money they are asking for and to be spent on what. For example $10k-$15k in cloud credits over a year to spend on building out a PoC to test out <some set of things>
+        3. TI(s) drafts that includes:
+          - Problem Statement in the form below
+          - Proposed work and other work considered
+          - Benefit to OSSF and ecosystem
+          - Approximate cost
+        4. TI submits this issue to the TAC. This will be treated like a CFP. Respondents will be notified in writing, and approved proposals will be published. For the first year, grants will be reviewed as they come in and will be reviewed until funding runs out for all qualified grant applications. 
```

Comment:
I'd like to advocate for a time-bound approach, where perhaps once per quarter requests are batched up and reviewed relative to one another. The FIFO model has some downsides that I think we could/should avoid.

Reply:
agreed

Reply:
I do agree. 
Here is the orginal timeline extended into the future

- Mar 11-15      Get form online, test it, put our call for Proposals
- Mar 18-29      Proposal submission period 
- April 1-5         TAC proposal review (need a special session?)
- April 8-12       Notify accepted/non acceptance proposals in writing; begin funding conversations; 
- April 15          Publish accepted proposals online and announce with SOSS Community Day NA
- April 25	B&F committee does a reallocation for June BC meeting	
- May 16	GB Meeting
- June 3- 14	Proposal submission period
- June 1-5         TAC proposal review
- June 17-21 	TAC proposal review (need a special session?)
- Aug 1 		B&F committee
- Aug 15 	GB Meeting
- Sept 1 - 13	Proposal submission period
- Sept 19 	SOSS Community Day EU
- Nov 7		B&F committee
- Nov 18		GB Meeting

I think a general proposal period that might work in line with the budget cycles would be:

- The first two weeks of Jan, Mar, Jun, and Sept. 

I think this is what @marcelamelara was hoping for as well?

Reply:
Yes, this is ultimately where I was hoping we'd land. This looks like a reasonable, and I agree with Bob that 4 days may be a bit tight for TAC review. I'm also wondering if we should also have a short "revision" period scheduled after the TAC review for requests that aren't clear accepts/rejects?